support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -50,17 +48,39 @@ class SendAcquisitionEvent(serviceProvider: ServiceProvider = ServiceProvider)
 
   }
 
+  private def buildGaData(state: SendAcquisitionEventState, requestInfo: RequestInfo): Either[String, GAData] = {
+    import cats.syntax.either._
+    for {
+      acquisitionData <- Either.fromOption(state.acquisitionData, "acquisition data not included")
+      ref = acquisitionData.referrerAcquisitionData
+      hostname <- Either.fromOption(ref.hostname, "missing hostname in referrer acquisition data")
+      gaClientId = ref.gaClientId.getOrElse(UUID.randomUUID().toString)
+      ipAddress = ref.ipAddress
+      userAgent = ref.userAgent
+    } yield GAData(
+      hostname = hostname,
+      clientId = gaClientId,
+      clientIpAddress = ipAddress,
+      clientUserAgent = userAgent
+    )
+  }
+
   private def sendAcquisitionEvent(state: SendAcquisitionEventState, requestInfo: RequestInfo, services: Services) = {
     sendPaymentSuccessMetric(state)
 
     val acquisition = AcquisitionDataRowBuilder.buildFromState(state, requestInfo)
 
     val streamFuture = services.acquisitionsStreamService.putAcquisitionWithRetry(acquisition, maxRetries = 5)
     val biqQueryFuture = services.bigQueryService.tableInsertRowWithRetry(acquisition, maxRetries = 5)
+    val gaFuture = for {
+      gaData <- EitherT.fromEither(buildGaData(state, requestInfo)).leftMap(err => List(err))
+      result <- services.gaService.submit(acquisition, gaData, maxRetries = 5).leftMap(gaErrors => gaErrors.map(_.getMessage))
```

Comment:
we had no idea at the time, but it turns out the acquisition from the `AcquisitionDataRowBuilder` labels are subtley different in terms of upper/lower case from the ones in SendOldAcqisitionState buildLabels function, which broke the gift reporting in GA as it checks the exact string
https://github.com/guardian/support-frontend/blob/13b0de701b1f7b6ef81a8b6ea61df8701f9b3211/support-modules/acquisition-events/src/main/scala/com/gu/support/acquisitions/ga/GoogleAnalyticsService.scala#L95